commit 7d421209287a07db5e926552ae5fbe9d8abb50dc
Author: Florian Weimer <fweimer@redhat.com>
Date:   Tue Jun 6 11:39:06 2023 +0200

    pthreads: Use _exit to terminate the tst-stdio1 test
    
    Previously, the exit function was used, but this causes the test to
    block (until the timeout) once exit is changed to lock stdio streams
    during flush.

diff --git a/sysdeps/pthread/tst-stdio1.c b/sysdeps/pthread/tst-stdio1.c
index 80fb59c4e42ca550..a2cc71d67f0761f6 100644
--- a/sysdeps/pthread/tst-stdio1.c
+++ b/sysdeps/pthread/tst-stdio1.c
@@ -47,7 +47,7 @@ do_test (void)
       _exit (1);
     }
 
-  delayed_exit (1);
+  delayed__exit (1);
   xpthread_join (th);
 
   puts ("join returned");
